spec/Client/HttpExceptionHandlerSpec.php

```diff
@@ -122,14 +222,34 @@ function it_throws_bad_request_exception_when_status_code_422(
             ->during('transformResponseToException', [$request, $response]);
     }
 
+    function it_throws_bad_request_exception_when_status_code_429(
```

Comment:
could you test the getter methods? Are you sure it works with "X-Retry-In: 110.5ms"

Reply:
X-Retry-In support dropped.
Added unit test for `getRetryAfter()` since it won't be called automatically